src/cli/build.rs

```diff
@@ -28,7 +29,13 @@ pub struct BuildCommand {
     ///
     /// Should end in .rbxm, .rbxl, .rbxmx, or .rbxlx.
     #[clap(long, short)]
-    pub output: PathBuf,
+    pub output: Option<PathBuf>,
+
+    /// Alternative to the output flag that outputs the result in the local plugins folder.
+    ///
+    /// Should end in .rbxm, .rbxl, .rbxmx, or .rbxlx.
```

Comment:
```suggestion
    /// Should end in .rbxm or .rbxmx.
```
It seems like bad UX to suggest building a place file to the plugins folder. Preventing someone from building a place file as a model and vice versa can come in a different PR (since it impacts the `build` command as a whole) but I think we should at least discourage it here.